docs/design/README.md

```diff
@@ -1,50 +1,63 @@
-# TiDB Design Documents
+# TiDB Improvement Proposals
 
-## Why We Need Design Documents
+This page describes a proposed TiDB Improvement Proposal process for proposing a major change to TiDB projects.
 
-The design document provides a high-level description of the architecture and important details of what we want to do. It is the most powerful tool to ensure that our work is correctly performed.
+## What is the purpose of improvement proposals?
 
-Writing a design document can promote us to think deliberately and gather knowledge from others to get our job done better. An excellent design document is closely related to the success of our task.
+The purpose of improvement proposals is to have a central place to collect and document planned major enhancements to TiDB projects. While GitHub Issue is still the tool to track tasks, bugs, and progress, the proposals give an accessible high level overview of the result of design discussions and proposals.
 
-## Proposal Process
+## Who should initiate the proposal?
 
-1. Before starting to write a design document, please [create a new issue](https://github.com/pingcap/tidb/issues/new/choose) for tracing the process of your design.
-2. Create a new [Google Doc](https://docs.google.com/document/u/0/) to write the design document. Please refer to this [proposal template](./TEMPLATE.md).
-3. If the design document is ready to be reviewed, please put the shared link with the `can comment` permission in the issue you've created, and one of our engineers will follow up this issue and keep it updated.
-4. If needed, we may organize a communications seminar through [Google Hangouts](https://hangouts.google.com/) to discuss your design with you and the interested friends from the community.
-5. When your design is finalized, please submit a pull request (PR) to add your new file under this directory, and put the link of your PR in the issue you've created. 
-6. Once your PR has been merged, please close the old issue.
-7. Start the implementation referring to the proposal, and create a new issue to trace the process.
+Anyone can initiate a proposal but you shouldn't do it unless you have an intention of getting the work done to implement it (otherwise it is silly).
```

Comment:
```suggestion
Anyone can initiate a proposal but you shouldn't do it unless you have an intention of getting the work done to implement it.
```

Reply:
Ok. Though I don't think it is offensive because that's the fact and we should emphasize it.